entries/SWC-124.md

```diff
@@ -0,0 +1,26 @@
+# Title
+Write to Arbitrary Storage Location
+
+## Relationships
+
+[CWE-123: Write-what-where Condition](https://cwe.mitre.org/data/definitions/123.html)
+
+## Description
+
+A smart contract's data (e.g., storing the owner of the contract) is persistently stored
+at some storage location (i.e., a key or address) on the EVM level. The contract is
+responsible for ensuring that only authorized clients may write to sensitive storage

```

Comment:
> clients
I would suggest accounts or users. Clients might be confusing.

Reply:
I've address this.